### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by Maksym-Lizohub**

### CODE REVIEW

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 1]
```diff
@@ -464,4 +464,217 @@
...
+                form {
+                    width: 100%;
+                }
+            }
+            .newsletter__sub-title {
+                width: 100%;
+            }
+            .lib-vendor-prefix-display(flex);
+            justify-content: space-between;
+            align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use mixin for align items

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 2]
```diff
@@ -464,4 +464,217 @@
...
+            .field {
+                .control:before {
+                    display: none;
+                }
+            }
+        }
+        .block.newsletter {
+            .form.subscribe {
+                display: flex;
+                align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use mixin for `display: flex` and `align-items: center`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 3]
```diff
@@ -464,4 +464,217 @@
...
+            margin-bottom: 40px;
+            .lib-font-size(16);
+            font-weight: @font-weight__heavier;
+            line-height: 16px;
+            text-transform: uppercase;
+        }
+
+        .footer__links-thumb__sub-list__item {
+            display: flex;
+            align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use mixin for `display: flex `and `align-items: center`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 4]
```diff
@@ -464,4 +464,217 @@
...
+                border-radius: 50%;
+                background-color: @color-maksym__main-black;
+                transition: all 0.4s ease 0s;
+            }
+        }
+        .footer__links-thumb__sub-list__link {
+            position: absolute;
+            left: 0;
+            display: flex;
+            align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use mixin for `display: flex` and `align-items: center`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 5]
```diff
@@ -464,4 +464,217 @@
...
+            text-decoration: none;
+            color: @color-maksym__main-gray;
+            transition: all 0.4s ease 0s;
+        }
+
+        .footer-copyright__list {
+            display: flex;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use mixin for `display: flex`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 6]
```diff
@@ -464,4 +464,217 @@
...
+                transition: all 0.4s ease 0s;
+                &:hover,
+                &:focus {
+                    text-decoration: none;
+                    color: @color-maksym__main-black;
+                }
+            }
+        }
+        .footer-social {
+            display: flex;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use mixin for `display: flex`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 7]
```diff
@@ -464,4 +464,217 @@
...
+            justify-content: flex-end;
+            p {
+                padding-right: 30px;
+                text-transform: uppercase;
+                margin: 0;
+                color: @color-maksym__main-gray;
+            }
+        }
+        .footer-social__list {
+            display: flex;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use mixin for `display: flex`

File: app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml [hunk 8]
```diff
@@ -15,7 +15,40 @@
...
+            </container>
+        </referenceContainer>
+
+        <move element="form.subscribe" destination="wraper.newsletter" after="-" />
+
+
+        <!-- FOTTER -->
+
+        <referenceBlock name="footer_links" remove="true"></referenceBlock>
+        <referenceBlock name="copyright" remove="true"></referenceBlock>
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml:55]: These void tags can be without closing tags like:
`<referenceBlock name="footer_links" remove="true" />`

File: app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less [hunk 9]
```diff
@@ -0,0 +1,284 @@
...
+                + .control {
+                    input {
+                        scale: 1;
+                        position: absolute;
+                        left: 75%;
+                        width: 20%;
+                        padding: 20px;
+                        font-size: 14px;
+                        background-color: #222;
+                        color: #fff;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less:67]: Use variables and `.lib-font-size()`

File: app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less [hunk 10]
```diff
@@ -0,0 +1,284 @@
...
+        }
+
+        .control {
+            border-top: 0;
+            margin: 0;
+            padding: 0;
+        }
+
+        input {
+            font-size: 16px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less:90]: Use `.lib-font-size()`

File: app/design/frontend/Elogic/maksym/Magento_Checkout/templates/cart/minicart.phtml [hunk 11]
```diff
@@ -0,0 +1,65 @@
+<li data-block="minicart" class="minicart-wrapper">
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Checkout/templates/cart/minicart.phtml:1]: Can you please add annotations for all overridden templates? Something like:
```
/**
 * Magento Checkout - Minicart
 * Overridden to add svg icon and modal
 */
```

**Review by y-mantuliak — CHANGES_REQUESTED** [review]

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 1 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 2 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 3 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 4 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 5 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 7 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml (hunk 8 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml:55]: done

File: app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less (hunk 9 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less:67]: done

File: app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less (hunk 10 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_CatalogSearch/web/css/source/_module.less:90]: done

File: app/design/frontend/Elogic/maksym/Magento_Checkout/templates/cart/minicart.phtml (hunk 11 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Checkout/templates/cart/minicart.phtml:1]: done

**Comment by Maksym-Lizohub** [comment]
Basecly i did all mixins on next commit... I'm sorry

**Review by y-mantuliak — APPROVED** [review]

**Pull request closed without merging**